console_bot/address_book.py

```diff
@@ -234,6 +234,48 @@ def get_next_week_birthdays(self):
 
         return get_birthdays_per_week(result)
 
+    def add_email(self, name, email):
+        if name in self.data:
+            is_valid, message = self.data[name].add_email(email)
+            return message
+        else:
+            return f"Contact {name} not found."
```

Comment:
Maybe we can expand a bit the message and ask to add contact first. wdyt?
```suggestion
            return f"Contact {name} not found. Add it first to the contact book"
```